### PR TITLE
First Assignment

### PR DESCRIPTION
**Pull request by TMPLR11**
First assignment

### CODE REVIEW

File: Assignment1/DatasetDescriptions.csv [hunk 1]
```diff
@@ -1 +1,7 @@
-Your name; Your GitHub user; Dataset name; Dataset URL; Dataset brief description
```
Comment (ocorcho) [inline Assignment1/DatasetDescriptions.csv:1]: Do not remove this line, please.

File: Assignment1/DatasetDescriptions.csv [hunk 2]
```diff
@@ -1 +1,7 @@
...
+Joel Ramirez Moreno; TMPLR11; Malware Detection PE-Based Analysis Using Deep Learning Algorithm Dataset; https://figshare.com/articles/Malware_Detection_PE-Based_Analysis_Using_Deep_Learning_Algorithm_Dataset/6635642/1;
+Dataset contains 8970 malware and 1000 benign binaries files. Malware files which are divided into 5 types:
```
Comment (ocorcho) [inline Assignment1/DatasetDescriptions.csv:2]: And remove all these additional lines, so that we have a final joint dataset for the group.

**Review by ocorcho — CHANGES_REQUESTED** [review]
I have done a couple of comments. I would have preferred something more on public procurement, but no worries, this is allowed. It is an initial excuse to have you using git.

File: Assignment1/DatasetDescriptions.csv (hunk 2 above)
Comment (TMPLR11) [reply Assignment1/DatasetDescriptions.csv:2]: Do you mean everything on an unique line right?

Comment (ocorcho) [reply Assignment1/DatasetDescriptions.csv:2]: Yes, but no need to have such an extensive description. Something much shorter.

**Comment by TMPLR11** [comment]
Think it's correct now

**Pull request merged**